src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/category_axis.html

```diff
@@ -43,46 +51,60 @@
         ng-class="{ 'fa-caret-down': isCategoryAxisAdvancedOptionsOpen, 'fa-caret-right': !isCategoryAxisAdvancedOptionsOpen }"
         class="kuiIcon fa-caret-right kuiSideBarOptionsLink__caret"
       ></span>
-      <span class="kuiSideBarOptionsLink__text">
-        <span ng-show="!isCategoryAxisAdvancedOptionsOpen">
-          Show
-        </span>
-        <span ng-show="isCategoryAxisAdvancedOptionsOpen">
-          Hide
-        </span>
-        Advanced Options
-      </span>
+      <span
+        class="kuiSideBarOptionsLink__text"
+        ng-show="!isCategoryAxisAdvancedOptionsOpen"
+        i18n-id="kbnVislibVisTypes.controls.pointSeries.categoryAxis.showAdvancedOptionsVisibilityLabel"
```

Comment:
Why are we using `showAdvancedOptionsVisibilityLabel`? Does the `visibility` have a special meaning in this context? I would suggest using `categoryAxis.showAdvancedOptionsLabel` instead.